backUpMessages.sh

```diff
@@ -23,7 +23,7 @@ mkdir $contactNumber
 cd $contactNumber
 #Perform SQL operations
 sqlite3 ~/Library/Messages/chat.db "
-select is_from_me,text, datetime(date + strftime('%s', '2001-01-01 00:00:00'), 'unixepoch', 'localtime') as date from message where handle_id=(
+select is_from_me,text, datetime((date / 1000000000) + strftime('%s', '2001-01-01 00:00:00'), 'unixepoch', 'localtime') as date from message where handle_id=(
```

Comment:
This would break the script for any OS < 10.13. Is there a way to check which MacOS is running from .sh?

Reply:
Agreed! I left a note about this at the bottom of my commit + PR. 
I believe `sw_vers -productVersion` can help here!